test/register_cluster1_cluster2_with_ocm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "Starting OCM registration"
+curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+
+export CLUSTER=cluster1
+kind create cluster --name ${CLUSTER}
+kubectl config rename-context kind-${CLUSTER} ${CLUSTER}
+clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${CLUSTER}/" | awk '{print $0 " --context '${CLUSTER}' --force-internal-endpoint-lookup"}' | sh
+
+export CLUSTER=cluster2
+kind create cluster --name ${CLUSTER}
+kubectl config rename-context kind-${CLUSTER} ${CLUSTER}
+clusteradm --context imbs1 get token | grep '^clusteradm join' | sed "s/<cluster_name>/${CLUSTER}/" | awk '{print $0 " --context '${CLUSTER}' --force-internal-endpoint-lookup"}' | sh
+
+kubectl config use-context imbs1
+
+echo sleep 40
+sleep 40
```

Comment:
Please let's not waste contributor time on dead-reckoning sleeps. Instead let's use a bash loop that tests for the presence of the expected objects.

Reply:
Removed the sleep in favor of a loop.